public class ArithmeticException
{
	public static void main(String args[])
	{
		int num1, num2; 

		try
		{
			int num1 = 100;
			Syste.out.println(a);
			int num2 = num1 / 0;
			num1 = num1 + 1;
			System.out.println(num1);
		}
		catch(ArithmeticException e)
		{
			System.out.println("Exception thrown " + e);
		}
	}
	 /* Author @raiprogramming - https://github.com/raiprograming */ 
}